pymess/models/emails.py

```diff
@@ -248,3 +278,20 @@ class EmailTemplateDisallowedObject(BaseRelatedObject):
     class Meta(BaseRelatedObject.Meta):
         verbose_name = _('disallowed object of an e-mail template')
         verbose_name_plural = _('disallowed objects of e-mail templates')
+
+
+class EmailTemplateStaticAttachment(SmartModel):
+
+    template = models.ForeignKey(settings.EMAIL_TEMPLATE_MODEL, verbose_name=_('template'),
+                                 on_delete=models.CASCADE, related_name='static_attachments')
+    file = models.FileField(verbose_name=_('file'), null=False, blank=False,
+                            upload_to=generate_static_attachment_filename)
+    send_as_filename = models.CharField(verbose_name=_('send as'), blank=True, null=True, max_length=100)
```

Comment:
in the Attachment is this stored into file name. Maybe will be better store file with the right filename. Yes you will have problem with the same files. But you change directory (for example with uuid) for every attachment file

for example
Path(settings.EMAIL_STORAGE_PATH) / 'attachments' / uud4() / filename

Reply:
maybe will be good add this solution in generate_attachment_filename and use the samo method here